### PR TITLE
Update CSS to use GOV.UK elements provisions

### PR DESCRIPTION
**Pull request by cruddasj**
## Description

Remove inline styling and separate out custom css vs. that provisioned in the GOV.UK template.

## Todo's before PR

- [X] Rebase from develop
- [X] Unit tests passed (`make test`)
- [X] PR naming according our [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [X] PR description describes the overall goals of PR commits
- [X] Templates been checked with relevant user-story

## Migrations

- [X] Null fields in models specifies default value
- [X] You generated and applied migrations (`make migrate`)
- [X] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [X] Specified reviewers (even if it's yourself)
- [X] Specified assigneses (those who'll merge)
- [X] Specified labels
- [ ] Specified milestone

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [X] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

**Comment by cruddasj** [comment]
All, please note that any custom css should be applied in custom.css going forward (please leave base.css as-is, under no circumstances adjusting its contents).

File: arc_application/templates/declaration-summary.html [hunk 1]
```diff
@@ -1074,11 +1074,6 @@ <h1 class="form-title heading-large">
             <input type="hidden" value="{{application_id}}" name="id"/>
         </form>
     </div>
-
-    <a href="{{ URL_PREFIX }}/review?id={{application_id}}" class="link-back"
```
Comment (ricaduchateau) [inline arc_application/templates/declaration-summary.html]: Is there no longer a return to list link for the master summary page in ARC?

File: arc_application/templates/personal-details-summary.html [hunk 2]
```diff
@@ -7,437 +7,409 @@
...
+                    </td>
+                    <td>
+                        {% if previous_registration %}
+                            Yes
+                        {% else %}
+                            No
+                        {% endif %}
+                    </td>
+                    <td class="change-answer">
+                        <a href="{{ URL_PREFIX }}/personal-details/previous-registration/?id={{application_id}}">Change</a>
```
Comment (ricaduchateau) [inline arc_application/templates/personal-details-summary.html]: Perhaps {% url '...' %} should be used here?

File: arc_application/templates/personal-details-summary.html [hunk 3]
```diff
@@ -7,437 +7,409 @@
...
-                        {{ name.first_name }}  {{ name.middle_names }} {{ name.last_name }}
+                        {% if individual_id %}
+                            {{ individual_id }}
+                        {% else %}
+                            None
+                        {% endif %}
                     </td>
                     <td class="change-answer">
-                        <a href="{% url 'personal_details_previous_names' %}?id={{ application_id }}&person_id={{ application_id }}&type=APPLICANT&referrer=list" alt="Change the {{ forloop.counter|inflect }} previous name">Change</a>
+                        <a href="{{ URL_PREFIX }}/personal-details/previous-registration/?id={{application_id}}">Change</a>
```
Comment (ricaduchateau) [inline arc_application/templates/personal-details-summary.html]: Are we no longer using {% url '...' %} to refer to the URL?

File: arc_application/templates/personal-details-summary.html [hunk 4]
```diff
@@ -7,437 +7,409 @@
...
+                    </td>
+                    <td>
+                        {% if five_years_in_UK %}
+                            Yes
+                        {% else %}
+                            No
+                        {% endif %}
+                    </td>
+                    <td class="change-answer">
+                        <a href="{{ URL_PREFIX }}/personal-details/previous-registration/?id={{application_id}}">Change</a>
```
Comment (ricaduchateau) [inline arc_application/templates/personal-details-summary.html]: Are we no longer using {% url '...' %} to refer to the URL?

**Pull request merged**